CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ These are just guidelines, not rules. Use your best judgment, and feel free to p
 
 This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
 By participating, you are expected to uphold this code.
-Please report unacceptable behavior to [team@aptly.info](mailto:team@aptly.info).
+Please report unacceptable behavior to [aptlypublic@gmail.com](mailto:aptlypublic@gmail.com).
```

Comment:
We don't have access to this mailing ? 

Should we not instead encourage the GitHub Discussion features instead ?